noir-projects/noir-contracts/contracts/docs_example_contract/src/main.nr

```diff
@@ -178,7 +178,7 @@ contract DocsExample {
     }
 
     #[aztec(private)]
-    fn insert_notes(amounts: [u8; 10]) {
+    fn insert_notes(amounts: [u8; 3]) {
```

Comment:
I reduced this value (see corresponding test - `note_getter`) to avoid hitting the enc logs limit. I'm happy to also increase the limit, but this does seem to slow down computation.